lib/webpack.js

```diff
@@ -132,3 +132,23 @@ exportPlugins(exports.node = {}, {
 exportPlugins(exports.debug = {}, {
 	"ProfilingPlugin": () => require("./debug/ProfilingPlugin"),
 });
+
+const defineMissingPluginError = (pluginName, errorMessage) => {
+	Reflect.defineProperty(exports.optimize, pluginName, {
```

Comment:
Not available in node 6. Could you use `Object.defineProperty` instead?

Reply:
Reflect actually does appear to be in node 6. I used NVM to test:
```
PS C:\Users\kingd> nvm install 6.0.0
Downloading node.js version 6.0.0 (64-bit)...
Complete
Creating C:\ProgramData\nvm\temp

Downloading npm version 3.8.6... Complete
Installing npm v3.8.6...

Installation complete. If you want to use this version, type

nvm use 6.0.0
PS C:\Users\kingd> nvm use 6.0.0
Now using node v6.0.0 (64-bit)
PS C:\Users\kingd> node -v
v6.0.0
PS C:\Users\kingd> node
> Reflect
Reflect
{}
> Reflect.defineProperty
[Function: defineProperty]
```

Reply:
Reflect is preferred since it doesn't throw like `Object.defineProperty` does.

Reply:
Nevermind. When zoomed in, node.green table is messed up.